### PR TITLE
Rl elasticsearch

### PR DESCRIPTION
**Pull request by RLuckom**

### CODE REVIEW

File: profiles/elasticsearch/terraform/elasticsearch/service.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  instance_data_disk_size = "${var.instance_data_disk_size}"
+  instance_data_disk_type = "${var.instance_data_disk_type}"
+  instance_data_disk_name = "${var.service}-data-disk"
+  instance_service_account = "${data.google_service_account.config_reader.email}"
+  instance_network_name = "${data.google_compute_network.terra-env-network.name}"
+  instance_labels = {
+    "app" = "${var.service}",
+    "owner" = "${var.owner}",
+    "role" = "frontend",
+    "ansible_branch" = "rl-add-services",
```
Comment (gmalkov) [inline profiles/elasticsearch/terraform/elasticsearch/service.tf]: master

File: profiles/elasticsearch/terraform/elasticsearch/variables.tf.ctmpl [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+#
+# - Common Vars
+#
+variable "instance_tags" {
+  default = [
+    "{{env "OWNER"}}-terra-{{env "SERVICE"}}",
+    "http-server",
+    "https-server",
+    "{{env "SERVICE"}}",
```
Comment (gmalkov) [inline profiles/elasticsearch/terraform/elasticsearch/variables.tf.ctmpl]: Add "mongo"?

Comment (RLuckom) [reply profiles/elasticsearch/terraform/elasticsearch/variables.tf.ctmpl]: this is elasticsearch, the service name is `elasticsearch`

File: profiles/elasticsearch/terraform/elasticsearch/service.tf (hunk 1 above)
Comment (RLuckom) [reply profiles/elasticsearch/terraform/elasticsearch/service.tf]: good catch will do

**Pull request merged**